test/tests/bootstrap/test_redfish10_os_bootstrap_base.py

```diff
@@ -42,22 +75,32 @@ def wait_for_task_complete(taskid, retries=60):
     print "Task failed with the following state: " + result['json']['TaskState']
     return False
 
+# download OS proxy config from host
+httpProxies = fit_common.rackhdapi('/api/2.0/config')['json']['httpProxies']
+# helper routine for selecting OS image path by matching proxy path
+def proxySelect(tag):
+    for entry in httpProxies:
+        if tag in entry['localPath']:
+            return entry['localPath']
+    return ''
+
 # ------------------------ Tests -------------------------------------
 from nose.plugins.attrib import attr
 @attr(all=True, regression=True)
 class os_bootstrap_base(fit_common.unittest.TestCase):
     def setUp(self):
         #delete active workflows for specified node
         fit_common.cancel_active_workflows(NODE)
-    #@fit_common.unittest.skipUnless('esxi55' in fit_common.GLOBAL_CONFIG['repos']['os'], "Skipping ESXi5.5")
-    @fit_common.unittest.skipUnless(False, "Skipping ESXi5.5")
+
+    @fit_common.unittest.skipUnless(proxySelect('ESXi/5.5') != '', "Skipping ESXi5.5")
     def test_bootstrap_esxi55(self):
         if fit_common.VERBOSITY >= 2:
             print 'Running ESXI 5.5 bootstrap.'
         nodehostname = 'esxi55'
-        payload_data = {"osName": "ESXi",
+        payload_data = {
+                        "osName": "ESXi",
                         "version": "5.5",
-                        "repo": fit_common.GLOBAL_CONFIG['repos']['os']['esxi55'],
+                        "repo": "http://172.31.128.1:9080" + proxySelect('ESXi/5.5'),
```

Comment:
I think so. That's a good idea since the southbound IP:port is configurable. I'll look into that.